code/modules/cargo/goodies.dm

```diff
@@ -31,37 +31,37 @@
 /datum/supply_pack/goody/combatknives_single
 	name = "Combat Knife Single-Pack"
 	desc = "Contains one sharpened combat knive. Guaranteed to fit snugly inside any Nanotrasen-standard boot."
-	cost = 800
+	cost = 1200
```

Comment:
just lower of the bulk price they are crappy spears

Reply:
the hell do you mean it literally does more damage than a spear and fits in jackboots

Reply:
and it makes it worth more than 3 stun batons? dude......... you can print a saw from autolathe for free....